example/example.config.js

```diff
@@ -63,11 +63,29 @@ const modules = [
     isResponsive: true
   }),
   colors({
+    nestedRules: {
+      ".dark {}": {
```

Comment:
keys are class templates, {} is substituted with the rule being generated (including a space generates children rules, without space a class join, etc...

the value is a new config for the current module, so we can change values and or whitelists and everything else